.github/workflows/reusable-audit.yml

```diff
@@ -0,0 +1,46 @@
+name: Reusable Audit Workflow
+
+on:
+  workflow_call:
+    inputs:
+      useForceResolutions:
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      WEBHOOK_E2E_FAILURE:
+        required: true
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
```

Comment:
```suggestion
```

Turns out we don't actually need the `npm ci` here, and we can just skip that. We may have previously had `npm ci` here when `license-checker` came from the `package.json`, but now it comes from npx, we don't need it.